intTests/test2049/Makefile

```diff
@@ -0,0 +1,12 @@
+CC = clang
+#CFLAGS = -g -frecord-command-line -O0
```

Comment:
Is -frecord-command-line important? My apple clang rejected it; I can get a non-apple-clang bytecode file if necessary but it's a bit of a nuisance.

Reply:
Oh dear. I had assumed that `-frecord-command-line` was portable, but it turns out that [it's not](https://github.com/GaloisInc/saw-script/issues/1157#issuecomment-2331287105). I certainly value making test cases portable when possible, so I don't think it's wise to use `-frecord-command-line` going forward. (We could engineer the `Makefile`s to only use `-frecord-command-line` on ELF targets, but that seems like more hassle than it's worth.)

Reply:
This is all to see: I think the way that `Makefile` is structured right now is fine, and I think you could delete this commented-out `CFLAGS` line.

Reply:
Done in 557a56f563ed8affb0ed405643041ee7d6fe134f